device/pensando/arm64-elba-asic-flash128-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,12 @@
+{
+    "skip_thermalctld": false,
+    "skip_fancontrol": true,
+    "skip_ledd": true,
+    "skip_psud": true,
+    "skip_syseepromd": false,
+    "skip_xcvrd": true,
```

Comment:
@shanshri Skipped because no transceiver?

Reply:
Yes. there is no transceiver connected. DPU is connected to NPU directly through trace links.

Reply:
@shanshri why then its set to true in device/pensando/arm64-elba-asic-r0/pmon_daemon_control.json